core/src/trezor/lvglui/scrs/address.py

```
@@ -359,10 +394,10 @@ def __init__(self):
         self.addr_type = None
         self.prev_session_id = None
 
-    def get_chain_info(self, msg_type) -> dict:
+    def get_chain_info(self, name) -> dict:
```

Ruff (0.8.2)

Missing type annotation for function argument `name`

(ANN001)